worlds/sm/variaRandomizer/utils/utils.py

```diff
@@ -358,35 +358,35 @@ def convertParam(randoParams, param, inverse=False):
         return "random"
     raise Exception("invalid value for parameter {}".format(param))
 
-def loadRandoPreset(world, player, args):
+def loadRandoPreset(options, args):
     defaultMultiValues = getDefaultMultiValues()
     diffs = ["easy", "medium", "hard", "harder", "hardcore", "mania", "infinity"]
     presetValues = getPresetValues()
 
-    args.animals = world.animals[player].value
-    args.noVariaTweaks = not world.varia_tweaks[player].value
-    args.maxDifficulty = diffs[world.max_difficulty[player].value]
-    #args.suitsRestriction = world.suits_restriction[player].value
-    args.hideItems = world.hide_items[player].value
-    args.strictMinors = world.strict_minors[player].value
-    args.noLayout = not world.layout_patches[player].value
-    args.gravityBehaviour = defaultMultiValues["gravityBehaviour"][world.gravity_behaviour[player].value]
-    args.nerfedCharge = world.nerfed_charge[player].value
-    args.area = world.area_randomization[player].current_key
+    args.animals = options.animals.value
+    args.noVariaTweaks = not options.varia_tweaks.value
+    args.maxDifficulty = diffs[options.max_difficulty.value]
+    #args.suitsRestriction = options.suits_restriction.value
+    args.hideItems = options.hide_items.value
+    args.strictMinors = options.strict_minors.value
+    args.noLayout = not options.layout_patches.value
+    args.gravityBehaviour = defaultMultiValues["gravityBehaviour"][options.gravity_behaviour.value]
+    args.nerfedCharge = options.nerfed_charge.value
+    args.area = options.area_randomization.current_key
     if (args.area == "true"):
         args.area = "full"
     if args.area != "off":
-        args.areaLayoutBase = not world.area_layout[player].value
-    args.escapeRando = world.escape_rando[player].value
-    args.noRemoveEscapeEnemies = not world.remove_escape_enemies[player].value
-    args.doorsColorsRando = world.doors_colors_rando[player].value
-    args.allowGreyDoors = world.allow_grey_doors[player].value
-    args.bosses = world.boss_randomization[player].value
-    if world.fun_combat[player].value:
-        args.superFun.append("Combat")
-    if world.fun_movement[player].value:
+        args.areaLayoutBase = not options.area_layout.value
+    args.escapeRando = options.escape_rando.value
+    args.noRemoveEscapeEnemies = not options.remove_escape_enemies.value
+    args.doorsColorsRando = options.doors_colors_rando.value
+    args.allowGreyDoors = options.allow_grey_doors.value
+    args.bosses = options.boss_randomization.value
+    if options.fun_combat.value:
+        args.superFun.append()
```

Comment:
I think this was removed accidentally
```suggestion
        args.superFun.append("Combat")
```